spec/spec_helper.rb

```diff
@@ -3,13 +3,6 @@
 # Use a consistent test timezone
 ENV["TZ"] = "UTC"
 
-SimpleCov.start do
```

Comment:
I think we should go the other way with this to match [what we've decided in `rails-template`](https://github.com/ackama/rails-template/pull/300) - while we're at it I think it would be good to also update our config to match the latest [config in `rails-template`](https://github.com/ackama/rails-template/blob/main/variants/backend-base/spec/spec_helper.rb#L4-L38) (with the required branch coverage being set to whatever it is currently + a todo saying we'd like to get it to 80)